frontend/src/components/Searchbar/Searchbar.tsx

```diff
@@ -24,21 +25,44 @@ export interface SearchbarProps extends InputProps {
   onSearch: (searchValue: string) => void
 
   /**
-   * Whether the searchbar is expanded or not.
-   * @note This should be `true` if the `onSearchIconClick` function prop is not
-   * provided, or the searchbar will not be usable.
+   * Whether the searchbar is expandable/collapsable or not. Defaults to `true`.
    */
-  isExpanded: boolean
+  isExpandable?: boolean
 
   /**
-   * Optional. Function to be invoked when the search icon is clicked.
-   * If provided, the search icon will be clickable.
+   * Whether the searchbar is initially expanded or not. Defaults to `false`.
+   * @note Set this to `true` if `isExpandable` is set to `false`, else the
```

Comment:
From a prop input contract perspective, I reckon if `isExpandable` is `false`, the `searchbar` should always be usable, whether `isInitiallyExpanded` is `true`, `false`, or not supplied at all 🤔 (reduce issue with callers making a mistake).

Reply:
Good point! Will update the component to reflect this.